.github/workflows/inductor-perf-max-autotune-weekly.yml

```diff
@@ -0,0 +1,45 @@
+name: inductor-A100-max-autotune-weekly
```

Comment:
Do you also need to add `inductor-A100-max-autotune-weekly` to https://github.com/pytorch/pytorch/blob/main/.github/workflows/upload-torch-dynamo-perf-stats.yml#L5 to get the performance results sent to Rockset?

Reply:
Yes, we will need that, but it might need some twist on the UI. Weekly dots vs. daily lines.

Reply:
@huydhn , I will leave the upload script as is for now and let's work out the details on how to distinguish the data from nightly and weekly.